cmd/antrea-agent/agent.go

```diff
@@ -504,6 +536,7 @@ func run(o *Options) error {
 			enableBridgingMode,
 			enableAntreaIPAM,
 			o.config.DisableTXChecksumOffload,
+			mtuDeduction,
```

Comment:
Partially Done, I added a new `MTUDeduction` to NetworkConfig and pass it to CNIServer. For current change, there is no need to get interface MTU since we can get the value from container's interface attributes directly for policy only mode. I feel it's better to add the new InterfaceMTU field and below changes in another PR:
```
In pkg/agent/agent.go, we can change func getNodeMTU() to calculatePodMTU() which calculates and sets MTUDeduction and InterfaceMTU of config.NetworkConfig.

We may further remove NodeConfig.NodeMTU and change setupGatewayInterface() to use NetworkConfig.InterfaceMTU.
```
Let's me know what's your thoughts.

Reply:
I would still add `NetworkConfig.InterfaceMTU` and change `loadNetworkConfig()` to use that, and also remove `NodeConfig.NodeMTU`. The reason is that we want not two ways to decide MTUs, which will be harder to maintain.